lms/db/__init__.py

```diff
@@ -4,16 +4,18 @@
 import alembic.config
 import sqlalchemy
 import zope.sqlalchemy
-from sqlalchemy import text
+from sqlalchemy import Select, text
+from sqlalchemy.dialects import postgresql
 from sqlalchemy.inspection import inspect
-from sqlalchemy.orm import declarative_base, sessionmaker
+from sqlalchemy.orm import Query, Session, declarative_base, sessionmaker
 from sqlalchemy.orm.properties import ColumnProperty
 
 from lms.db._columns import varchar_enum
 from lms.db._locks import CouldNotAcquireLock, LockType, try_advisory_transaction_lock
 from lms.db._text_search import full_text_match
+from lms.db._util import compile_query
 
-__all__ = ("Base", "create_engine", "varchar_enum")
+__all__ = ("Base", "compile_query", "create_engine", "varchar_enum")
```

Comment:
What's with all the changed imports here?

Reply:
compile_query belongs on __all__ IMO, unless we remove __all__completetly.